src/rules/code-examples/banTsIgnore.examples.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright 2018 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Lint from "../../index";
+
+// tslint:disable: object-literal-sort-keys
+export const codeExamples = [
+    {
+        description: 'Disallows the use of "@ts-ignore"',
+        config: Lint.Utils.dedent`
+            "rules": { "ban-ts-ignore": true }
+        `,
+        pass: Lint.Utils.dedent`
+            console.log("hello");
```

Comment:
I think use the same code snippet from the `fail` example, but change the comment text to
```ts
// Compiler warns about unreachable code error
```

Reply:
Done. Thanks!